thewalrus/quantum.py

```diff
@@ -788,3 +821,77 @@ def total_photon_num_dist_pure_state(cov, cutoff=50, hbar=2, padding_factor=2):
         rs = np.arctanh(np.linalg.svd(B, compute_uv=False))
         return gen_multi_mode_dist(rs, cutoff=cutoff, padding_factor=padding_factor)[0:cutoff]
     raise ValueError("The Gaussian state is not pure")
+
+
+
+

```

Comment:
Too many line breaks -- don't forget to `black -l 100 your/files.py` before merging!

Reply:
Will do!